### DISCUSSION TITLE
Simil "code-coverage" feature

### DISCUSSION THREAD

**Discussion by MarcoFierimonte** [apideck-libraries/portman#347] [Ideas]
Hi :) 

I was thinking that would be great have a code coverage parameters for portman generated test. 
The code coverage could be calculated like "The number of API testing executed with success respect to the full OpenAPI". 
This params can be computed with/without considering the portman "api-filter.json" file too. 

Thanks for this great tool.
Marco F.

**Comment by thim81** [comment]
hi @MarcoFierimonte,

Thanks to @NickHeap2, there is an Newman extension that does just that.
https://github.com/NickHeap2/newman-reporter-openapi

The power of Opensource.

**Reply by MarcoFierimonte** [reply]
Hi @thim81 ,

thanks for info, I'll try this tool asap :) 

Greetings.
Marco F:

**Reply by NickHeap2** [reply]
@MarcoFierimonte any feedback or suggestions feel free to ping me :)

**Reply by MarcoFierimonte** [reply]
thanks @NickHeap2  :)
